lib/MusicBrainz/Server/Validation.pm

```diff
@@ -317,6 +317,9 @@ sub is_valid_edit_note
 
     return 0 unless $edit_note;
 
+    # We don't want line format chars to stop an edit note from being "empty"
+    $edit_note =~ s/[\N{ZERO WIDTH SPACE}\N{SOFT HYPHEN}\p{Cc}\p{Cf}\p{Mn}]//g;
```

Comment:
Can this use `remove_lineformatting_characters` for one step and then strip the extra stuff in another step? Just in case we update `remove_lineformatting_characters`in the future; I'd expect we'd want the same changes propagated here.

Reply:
Ok, but then I also need to keep a note to keep that one in sync with `invalidEditNote` :) Done.